Source/Atomic/Engine/Engine.cpp

```diff
@@ -379,6 +379,9 @@ bool Engine::Initialize(const VariantMap& parameters)
         ))
             return false;
 
+        if (GetParameter(parameters, "WindowMaximized", false) == true)
+            graphics->Maximize();

```

Comment:
This should be factored into the SetMode call immediately above:

https://github.com/rsredsq/AtomicGameEngine/blob/RED-EDITOR-511/Source/Atomic/Engine/Engine.cpp#L370

That way the window isn't created and then immediately resized for maximize